hands-on-exercise/1-native/index.md

```diff
@@ -0,0 +1,57 @@
+---
+parent:
+  title: First Steps
+  description: Get started with Cosmos SDK v0.50 and run your Cosmos chain
+  number: 1
+order: 0
+title: Chapter Overview - First Steps to Run Your Chain
+tags:
+---
+
+<div class="tm-overline tm-rf-1 tm-lh-title tm-medium tm-muted">Get started with Cosmos SDK v0.50 and run your Cosmos chain</div>
```

Comment:
If we want to keep this title, I believe we should change the one in `1-ignite-cli` to more strongly differentiate between the two chapters.